Octokit.Reactive/Clients/IObservableIssueTimelineClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Octokit.Reactive
+{
+    /// <summary>
+    /// A client for GitHub's Issue Timeline API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="http://developer.github.com/v3/issues/timeline/">Issue Timeline API documentation</a> for more information.
```

Comment:
That's very odd as I'm pretty sure I copied those from the API documentation...I'll fix it and check the docs if there might something pointing to just `http`